### PR TITLE
Makes ectoscopic sniffer more obvious, plus fix

### PR DESCRIPTION
**Pull request by mystery3525**
## About The Pull Request

Most roboticists who do not lurk on github or view change log, and do not know the sniffer even exists, much less whatever the hell that beeping noise in robotics is. This should make it more obvious.

## Why It's Good For The Game

Fixed the wrench_act proc for the sniffer to use default_unfasten_wrench (shouldn't this be more universal?), as well as making it more obvious for roboticsts, doubling the delay to 30 seconds to compensate.

As per deadchat advice, move the sniffer in front of the circuit printer for most (corg is fine) maps so it is not covered by clutter.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: More info messages for ectoscopic sniffer.
fix: Ectoscopic sniffer wrenching.
/:cl:

### CODE REVIEW

File: code/game/machinery/ecto_sniffer.dm [hunk 1]
```diff
@@ -19,21 +19,23 @@
 	wires = new/datum/wires/ecto_sniffer(src)
 
 /obj/machinery/ecto_sniffer/attack_ghost(mob/user)
-	. = ..()
+	///don't call ..() due to special ghost functionality
```
Comment (Dejaku51) [inline code/game/machinery/ecto_sniffer.dm]: maybe just remove that line?

Comment (mystery3525) [reply code/game/machinery/ecto_sniffer.dm]: Fair

**Review by Dejaku51 — DISMISSED** [review]
Well.. I approve that PR

**Comment by mystery3525** [comment]
Would it also make sense to do a `is_banned_from()` checks, same as posis, inside of the sniffer too? This may prevent banned players from being a nuisance.

**Comment by Dejaku51** [comment]
> Would it also make sense to do a `is_banned_from()` checks, same as posis, inside of the sniffer too? This may prevent banned players from being a nuisance.

go for it

File: code/game/machinery/ecto_sniffer.dm [hunk 2]
```diff
@@ -52,9 +57,7 @@
 
 
 /obj/machinery/ecto_sniffer/wrench_act(mob/living/user, obj/item/tool)
-	tool.play_tool_sound(src, 15)
-	setAnchored(!anchored)
-	balloon_alert(user, "sniffer [anchored ? "anchored" : "unanchored"]")
```
Comment (Archanial) [inline code/game/machinery/ecto_sniffer.dm:57]: balloon alerts were pretty cool here

File: code/game/machinery/ecto_sniffer.dm [hunk 3]
```diff
@@ -19,21 +19,26 @@
 	wires = new/datum/wires/ecto_sniffer(src)
 
 /obj/machinery/ecto_sniffer/attack_ghost(mob/user)
-	. = ..()
```
Comment (Archanial) [inline code/game/machinery/ecto_sniffer.dm:22]: What does parent do? Why did you remove it?

Comment (Dejaku51) [reply code/game/machinery/ecto_sniffer.dm:22]: examine iirc
so i'd say it is good to remove it

Comment (mystery3525) [reply code/game/machinery/ecto_sniffer.dm:22]: Looked at other ghost actions and saw that they didn't include it, this allows ghosts who have inquisitiveness enabled to still interact with the sniffer without an examine interaction. Doesn't stop regular examine.

File: code/game/machinery/ecto_sniffer.dm (hunk 2 above)
Comment (mystery3525) [reply code/game/machinery/ecto_sniffer.dm:57]: Opt'ed to use `default_unfasten_wrench` since it handles the wrench anchor/unanchor, else you hit the sniffer and toggle anchor each time; although, we could use balloon alerts in the `default_unfasten_wrench`.

EDIT: Turns out a lot of machines either do something custom for unanchoring, use unfasten_wrench, or something really weird, could make a new PR to make those more uniform

**Comment by Dejaku51** [comment]
i don't think changing their position to floor is good idea

**Comment by mystery3525** [comment]
I weighed the options, and unless we're adding a new table for each robotics in each map, there's no way we can move all the items on one table to other locations, to make them "fit in", plus the sniffer does not block moving (even when wrenched). Like, in Generic Map A, you got:
- Materials Table
- Toolboxes Table
- Cell Charger Table
- Spare Boards, Bodybags, and Gloves Table
- Spare Medibot parts Table
- Spare borg parts + flashes Table
- Wikibook + Boards(?) + Prox Sensors Table
- Figure, clipboard, paper Table

TL; DR: There just isn't enough space for the sniffer to have it's own table on all maps

**Review by PowerfulBacon — APPROVED** [review]
